src/DDTrace/Integrations/Mongo/MongoClientIntegration.php

```diff
@@ -0,0 +1,59 @@
+<?php
```

Comment:
Great work overall, it would be great if we could add some comments to the various methods and classes. :)